plugins/cpp/parser/src/clangastvisitor.h

```diff
@@ -600,7 +600,7 @@ class ClangASTVisitor : public clang::RecursiveASTVisitor<ClangASTVisitor>
 
     cppFunction->astNodeId = astNode->id;
     cppFunction->mangledNameHash = astNode->mangledNameHash;
-    cppFunction->name = getSignature(fn_);
+    cppFunction->name = astNode->astValue;
```

Comment:
Why do we store the signature multiple times: once in the `astValue` of the `CppAstNode` and second in the `name` attribute of the `CppFunction`.